plugins/techdocs/src/alpha.tsx

```diff
@@ -44,8 +68,89 @@ export const TechDocsSearchResultListItemExtension =
     },
   });
 
+/**
+ * Responsible for rendering the provided router element
+ *
+ * @alpha
+ */
+const TechDocsIndexPage = createPageExtension({
+  id: 'plugin.techdocs.indexPage',
+  defaultPath: '/docs',
+  routeRef: rootRouteRef,
+  loader: () =>
+    import('./home/components/TechDocsIndexPage').then(m => (
+      <m.TechDocsIndexPage />
+    )),
+});
+
+/**
+ * Component responsible for composing a TechDocs reader page experience
+ *
+ * @alpha
+ */
+const TechDocsReaderPage = createPageExtension({
+  id: 'plugin.techdocs.readerPage',
+  loader: () =>
+    import('./reader/components/TechDocsReaderPage').then(m => (
+      <m.TechDocsReaderPage />
```

Comment:
Suggestion (non-blocking):
Do addons fall under this pull request? It might be a follow-up pull request, but when thinking about how adopters could install `TechDocs` addons, a suggestion is to inject them as inputs to the reader's page 🙂

Reply:
I don't think so for now. Should I create a seperate issue?

Reply:
I would recommend creating a separate issue, it will be a good exercise to test the extensions framework 🙌🏻